src/scss/src/_variables.scss

```diff
@@ -151,9 +153,41 @@ $spacing-extra-large: $spacing-base * 3 !default;
 $border: 1px solid color('gray', 'lightest') !default;
 $border-radius-main: 3px !default;
 $border-radius-medium: 6px !default;
+$border-radius-full: 9999px !default;
 $border-radius-progress-bar: 9px !default;
 $ul-ol-padding-left: $spacing-base !default;
 
+/**
+ * Border radius
+ */
+
+$border-radius-base: 3px !default;
+
+$border-radiuses: (
+  0: 0,                         // 0px
+  'base': $border-radius-base,  // 3px
+  1: $border-radius-base * 2,   // 6px
+  'full': 50%                   // 50%
+) !default;
+
+
+/**
+ * Display
+ */
+
+$displays: (
+  'block': block,
+  'inline-block': inline-block,
+  'inline': inline,
+  'flex': flex,
+  'inline-flex': inline-flex,
+  'table': table,
+  'table-row': table-row,
+  'table-cell': table-cell,
+  'grid': grid,
+  'hidden': none
```

Comment:
I think it's better that we follow the css properties and keep it `.d-none`, instead of giving it a totally different name. It's true that we all used `.hidden` in the past and got used to it, but with `.d-{key}` prefix, it doesn't make much sense.